CHANGELOG.md

```diff
@@ -0,0 +1,33 @@
+0.4.0 (2021-05-17)
+===
+
+* Fixes a bug where benchmarks got stock randomly due to a synchronisation issue
+* Removes duplicate edges
```

Comment:
It is actually a bug fix that caused duplicated edges.

Reply:
Do you mean duplicate edges caused the synchronisation bug?

Reply:
We push roots multiple times that caused the duplicate edges, which is a bug. This is what I meant.

Reply:
Right, I will change to reflect that